.editorconfig

```diff
@@ -94,9 +94,9 @@ csharp_style_expression_bodied_local_functions = false:none
 csharp_style_expression_bodied_lambdas = true:none
 
 # Prefer property-like constructs to have an expression-body
-csharp_style_expression_bodied_properties = true:none
-csharp_style_expression_bodied_indexers = true:none
-csharp_style_expression_bodied_accessors = true:none
+csharp_style_expression_bodied_properties = true:warning
+csharp_style_expression_bodied_indexers = true:warning
+csharp_style_expression_bodied_accessors = true:warning
```

Comment:
All style rules are defaulted to warnings, so this is redundant.
```suggestion
```

https://github.com/dotnet/roslyn-analyzers/blob/5102a0d89ef178b0ca741634ee006597bf5df2ed/.editorconfig#L188-L189